src/Dropdown.php

```diff
@@ -959,79 +959,85 @@ public static function dropdownUsedItemTypes($name, $itemtype_ref, $options = []
+                $values,
+                array_merge(
+                    [
+                        'value'                 => $value,
+                        'display_emptychoice'   => true,
+                        'display'               => $display,
+                        'noselect2'             => true, // we will instanciate it later
+                        'rand'                  => $rand,
+                    ],
+                    $options
+                )
+            );
 
-                /** @var array $CFG_GLPI */
-                global $CFG_GLPI;
+            /** @var array $CFG_GLPI */
+            global $CFG_GLPI;
 
-                // templates for select2 dropdown
-                $js = <<<JAVASCRIPT
-                $(function() {
-                    const formatFormIcon = function(icon) {
-                        if (!icon.id || icon.id == '0') {
-                            return icon.text;
-                        }
-                        var img = '<span><img alt="" src="{$CFG_GLPI['typedoc_icon_dir']}/'+icon.id+'" />';
-                        var label = '<span>'+icon.text+'</span>';
-                        return $(img+'&nbsp;'+label);
-                    };
-                    $("#dropdown_{$myname}{$rand}").select2({
-                        width: '60%',
-                        templateSelection: formatFormIcon,
-                        templateResult: formatFormIcon
-                    });
+            // templates for select2 dropdown
+            $js = <<<JAVASCRIPT
+            $(function() {
+                const formatFormIcon = function(icon) {
+                    if (!icon.id || icon.id == '0') {
+                        return icon.text;
+                    }
+                    var img = '<span><img alt="" src="{$CFG_GLPI['typedoc_icon_dir']}/'+icon.id+'" />';
+                    var label = '<span>'+icon.text+'</span>';
+                    return $(img+'&nbsp;'+label);
+                };
+                $("#dropdown_{$myname}{$rand}").select2({
+                    width: '60%',
+                    templateSelection: formatFormIcon,
+                    templateResult: formatFormIcon
                 });
+            });
 JAVASCRIPT;
-                echo Html::scriptBlock($js);
-            } else {
-               //TRANS: %s is the store path
-                printf(__('Error reading directory %s'), $store_path);
-            }
+            echo Html::scriptBlock($js);
         } else {
-           //TRANS: %s is the store path
-            printf(__('Error: %s is not a directory'), $store_path);
+            $error_msg = __('Error reading icon directory');
+            echo <<<HTML
+            <div class="alert alert-danger">
+                <span class="fs-4 alert-title">
+                    $error_msg
+                </span>
+            </div>
+HTML;
+
+            trigger_error(sprintf(__('Error reading directory %s'), $icon_path), E_USER_WARNING);
```

Comment:
Logged messages should not be translated. We want to have untranslated log messages when someone report a bug.

```suggestion
            trigger_error(sprintf('Error reading directory %s', $icon_path), E_USER_WARNING);
```